webextensions/api/menus.json

```
@@ -188,18 +198,21 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": true
+                  "version_added": true,
+                  "notes": "Available for use in Manifest V2 only."
                 },
                 "edge": "mirror",
                 "firefox": {
-                  "version_added": "53"
+                  "version_added": "53",
+                  "notes": "Available for use in Manifest V2 only."
```

Sorry for missing this one: Page action remains available in Firefox for MV3
```suggestion
                  "version_added": "53"
```